src/components/InputField/InputField.tsx

```diff
@@ -121,12 +124,17 @@ export type Props = React.InputHTMLAttributes<HTMLInputElement> & {
     }
   >;
 
+type InputFieldType = ForwardedRefComponent<HTMLInputElement, Props> & {
+  Input?: typeof Input;
+  Label?: typeof Label;
+};
+
 /**
  * `import {InputField} from "@chanzuckerberg/eds";`
  *
  * An input with optional labels and error messaging built-in.
```

Comment:
While here, can you add similar guidance about how the field must be labeled ([example](https://github.com/chanzuckerberg/edu-design-system/blob/next/src/components/TextareaField/TextareaField.tsx#L58))

Reply:
Yup, done